index.js

```diff
@@ -395,7 +397,9 @@ async function run() {
     const desiredCount = parseInt((core.getInput('desired-count', {required: false})));
     const enableECSManagedTagsInput = core.getInput('enable-ecs-managed-tags', { required: false }) || 'false';
     const enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
-   
+    const propagateTags = core.getInput('propagate-tags', { required: false }) || 'NONE';
+    const runTaskTags = JSON.parse(core.getInput('run-task-tags', { required: false }) || '[]');
```

Comment:
We can read the config related to runTask in the `runTask` function. So `runTaskTags` config can be read inside the `runTask` function like other configs starting from line 42.

Reply:
✅